hadoop-common-project/hadoop-common/pom.xml

```diff
@@ -147,6 +147,24 @@
       <groupId>com.sun.jersey</groupId>
       <artifactId>jersey-json</artifactId>
       <scope>compile</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.codehaus.jackson</groupId>
+          <artifactId>jackson-core-asl</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.codehaus.jackson</groupId>
+          <artifactId>jackson-mapper-asl</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.codehaus.jackson</groupId>
+          <artifactId>jackson-jaxrs</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.codehaus.jackson</groupId>
+          <artifactId>jackson-xc</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
This is required to avoid dependency convergence issue:

```
Dependency convergence error for org.codehaus.jackson:jackson-mapper-asl:1.9.2 paths to dependency are:
+-org.apache.hadoop:hadoop-common:3.4.0-SNAPSHOT
  +-com.sun.jersey:jersey-json:1.19
    +-org.codehaus.jackson:jackson-mapper-asl:1.9.2
and
+-org.apache.hadoop:hadoop-common:3.4.0-SNAPSHOT
  +-com.sun.jersey:jersey-json:1.19
    +-org.codehaus.jackson:jackson-jaxrs:1.9.2
      +-org.codehaus.jackson:jackson-mapper-asl:1.9.2
and
+-org.apache.hadoop:hadoop-common:3.4.0-SNAPSHOT
  +-com.sun.jersey:jersey-json:1.19
    +-org.codehaus.jackson:jackson-xc:1.9.2
      +-org.codehaus.jackson:jackson-mapper-asl:1.9.2
and
+-org.apache.hadoop:hadoop-common:3.4.0-SNAPSHOT
  +-org.apache.avro:avro:1.7.7
    +-org.codehaus.jackson:jackson-mapper-asl:1.9.13

```